iset.mm

```diff
@@ -82997,6 +83073,100 @@ Nonnegative integers (as a subset of complex numbers)
     $.
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+                              Extended nonnegative integers
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+  The function values of the hash (set size) function are either nonnegative
```

Comment:
Simply: "the set size function" ?

I do not know how standard it is, but I think calling the set size function the "hash" function because of the symbol generally denoting it, is bad practice (a bit like "the epsilon relation" for membership, or "the vertical bar letter" for I).

Reply:
I like "set size function". I think I've probably seen it in the literature somewhere but maybe just in set.mm. AczelRathjen seems to just say "This justifies that we call this number n the number of elements of A and denote it by ](A)."